bank_account_wave2a.rb

```diff
@@ -40,16 +41,24 @@ def deposit (deposit_amount)
     def self.all
```

Comment:
On lines 29-31 above, since you have an instance variable called `balance` and an attr_accessor for that instance variable on line 6, this method is not doing anything additional and can be removed.